lib/OrePAN2/Repository.pm

```diff
@@ -1,58 +1,44 @@
 package OrePAN2::Repository;
 
-use strict;
-use warnings;
 use utf8;
-use 5.008_001;
 
-use Carp ();
-use Class::Accessor::Lite 0.05 (
-    rw => [qw(directory cache compress_index)],
-);
+use Moo;
+
 use File::Find                 ();
 use File::Spec                 ();
 use File::pushd                ();
 use OrePAN2::Indexer           ();
 use OrePAN2::Injector          ();
 use OrePAN2::Repository::Cache ();
+use Types::Standard            qw( Bool InstanceOf Str );
 
-sub new {
-    my $class = shift;
-    my %args  = @_ == 1 ? %{ $_[0] } : @_;
+use namespace::clean;
```

Comment:
Any reason not to use `namespace::autoclean`?

Reply:
No not really. I am using namespace::clean because the Moo documentation provides examples how to work with it and scrubbing is restricted to anything imported before its use.